myapp/src/components/Discussion/Cards/index.js

```diff
@@ -1,95 +1,76 @@
 import React from 'react';
-import Card from 'react-bootstrap/Card'
-import Accordion from 'react-bootstrap/Accordion'
+import Card from 'react-bootstrap/Card';
+import Accordion from 'react-bootstrap/Accordion';
 import {
-  CardHoover,
-  UpdateText,
+  ThreadCard,
+  TitleText,
   DescriptionText,
   NavLink
 } from './CardElements';
 
-function Cards() {
-  const information = [
-    {
-      route: '/thread',
-      title: '📢Announcements',
-      description: 'Announcements/FAQs/Logistics',
-      threads: '🧵 2 Threads',
-      posts: '💬 5 Posts',
-      newest: '⏲️Meeting 2/19',
-    },
-    {
-      route: '/thread',
-      title: '✒️General',
-      description: 'General logisitics about the team and other information.',
-      threads: '🧵 2 Threads',
-      posts: '💬 5 Posts',
-      newest: '⏲️How To Meme',
-    },
-    {
-      route: '/thread',
-      title: '🤖Robot',
-      description: 'General discussion about the robots.',
-      threads: '🧵 2 Threads',
-      posts: '💬 5 Posts',
-      newest: '⏲️Robot Fails',
-    },
-    {
-      route: '/thread',
-      title: '⚔️Competition',
-      description: 'Talk about the competition here.',
-      threads: '🧵 2 Threads',
-      posts: '💬 5 Posts',
-      newest: '⏲️Ventura Regional 2021',
-    },
-  ];
+function Cards(props) {
+  const groupCards = props.groupData.map((group, index)  => {
+    const threads = group.threads.map((thread, index) => {
+      return (
+        <>
+          <ThreadCard>
+            <Card.Body>
+              <table>
+                <tr>
+                  <td>
+                    <NavLink to={"discussion/" + thread.url}>
+                      <TitleText>
+                        {thread.name}
+                      </TitleText>
+                    </NavLink>
+                    <DescriptionText>
+                      {thread.description}
+                    </DescriptionText>
+                  </td>
+                  <td>
+                    <button onClick={() => props.deleteThread({
+                              groupName: group.groupName,
+                              threads: [thread]
+                            })}>Delete</button>
+                  </td>
+                </tr>
+              </table>
+            </Card.Body>
+          </ThreadCard>
+        </>
+      );
+    });
 
-  const list = information.map((row, index)  => {
     return (
       <>
-        <Accordion>
-          <CardHoover key={ index }>
+        <Card>
```

Comment:
I like the simplicity of this. It makes it more condensed and specific to its file.